sources/packages/web/src/views/student/RestrictionBanner.vue

```diff
@@ -1,14 +1,59 @@
 <template>
-  <Message severity="error">
-    {{ restrictionMessage }}
-  </Message>
+  <banner
```

Comment:
`RestrictionBanner.vue` is a component, right? I believe that it must be placed inside the components.